atomic_scanners/pipeline-scanner/pipeline-scanner

```diff
@@ -1,10 +1,10 @@
 type: scanner
 scanner_name: pipeline-scanner
 image_name: registry.centos.org/pipeline-images/pipeline-scanner
-default_scan: redhat-release
+default_scan: pipeline-scanner
 scans: [
-  { name: redhat-release,
+  { name: pipeline-scanner,
```

Comment:
What I'm suggesting is to have a name like `yum-update` or `rpm-update`. Reason being that additional scan types we plan to include for the pipeline will come under the same scanner, that is, `pipeline-scanner`.  Naming the scan type to `pipeline-scanner` doesn't help one understand the purpose of the scan.

I understand that the way [scanner worker is currently designed](https://github.com/CentOS/container-pipeline-service/blob/master/beanstalk_worker/worker_start_scan.py#L346), this would break stuff. We need to add scan types to the command we're executing to get proper scan results. This can be done by passing `--scan-type=<name>` to the command. That should help take care of things when, in future, we add more scan types.

Also, at the moment, this won't break the worker because we've got only one scan type under the `pipeline-scanner` and it's default. So, `atomic scan` will execute that without us providing any scan type to it.